ballerina/caching_request_cache_control.bal

```diff
@@ -61,20 +61,20 @@ public class RequestCacheControl {
         }
 
         if self.maxAge >= 0d {
-            directives[i] = MAX_AGE + "=" + self.maxAge.toString();
+            directives[i] = MAX_AGE + "=" + decimal:floor(self.maxAge).toString();
```

Comment:
Can't we do this ?
```suggestion
            directives[i] = string `${MAX_AGE}=${decimal:floor(self.maxAge)}`;
```

Reply:
Fixed with [ca1ca4f](https://github.com/ballerina-platform/module-ballerina-http/pull/2085/commits/ca1ca4feadf1c0ed59dc22fa0b59c72d17981e47)